Src/motor_controller_functions.c

```diff
@@ -44,13 +44,8 @@ void mcCmdTorque(uint16_t torqueVal) {
 	tx.Data[0] = 	REGID_CMD_TORQUE;
 	tx.Data[1] =	(uint8_t) torqueVal;	//bytes 7-0
```

Comment:
I know this wasn't part of your change request but we should discuss how we do comments like this. I think it should be inline to the right, but having the different spacing between the comments for 45 and 46 bother me. I'll make a separate issue for this.

Reply:
Inline to the right makes the most sense to me.